src/editor/components/Controls/Button/index.js

```diff
@@ -10,7 +10,7 @@ const Button = ({ description, icon, onClick, active, disabled }: {
   description: string,
   icon: any,
   active: boolean,
-  disabled: boolean,
+  disabled?: boolean,
```

Comment:
Yes, in flow this means that the disabled property can be undefined (is optional). `disabled: ?boolean` would mean that disabled can be null, true or false